manifests/ssltools/certutil.pp

```diff
@@ -12,8 +12,8 @@
   } ->
   exec { $cert_name:
     path        => ['/bin', '/usr/bin'],
-    command     => "certutil -A -d '${nss_db_dir}' -n '${cert_name}' -t '${trustargs}' -a -i '${client_cert}'",
-    unless      => "certutil -L -d ${nss_db_dir} | grep '^${cert_name}\\b'",
+    command     => "certutil -A -d '${nss_db_dir}' -n '${cert_name}' -t '${trustargs}' -a -i '${client_cert}'  -f '${::certs::ssltools::nssdb::nss_db_password_file}'",
+    unless      => "certutil -L -d ${nss_db_dir}  -f '${::certs::ssltools::nssdb::nss_db_password_file}' | grep '^${cert_name}\\b'",
```

Comment:
I've tested this and `certutil -L` doesn't even have the `-f` option, and it doesn't need it: I suggest not providing it "just in case" , as it might be just misleading.

I've tested this and with the `-A` update, everything worked just fine